test cases/common/216 custom target input extracted objects/libdir/meson.build

```diff
@@ -18,4 +18,4 @@ subobjlib = static_library('subobject', sublibsrc)
 
 objlib = static_library('object', 'source.c', ctsrc, gensrc,
                         objects: subobjlib.extract_all_objects(recursive: false),
-                        override_options : ['unity=off'])
+                        override_options : {'unity': 'off'})
```

Comment:
Not sure I see the value of changing this test since you added a new one. I would say just leave it. Same for the other test you changed.

Reply:
The test case I added verify `default_options` but not `override_options`. But I agree it's better to do both in the same test, changed that.